### ISSUE TITLE
[css3-selectors] Create marketing.md

### ISSUE THREAD

**Issue by karlalopez** [gdisf/teaching-materials#445]
1. Create `marketing.md` in the lesson folder
2. Add content:

```
## One Paragraph Announcement

:tada: * CSS3 Selectors* :tada:
:calendar: Calendar Day, Month
:clock6: XX AM - XX PM @ Venue

Learn modern CSS3 selectors that will let you write more powerful, specific CSS rules. You'll strengthen your CSS rule writing muscles, helping you be more comfortable in the <style> tag.

We also have scholarships available.

*Sign up here:* [TODO: add link to meetup]

## Example Tweets
- Need to learn more about CSS attribute selectors, form pseudo-classes, structural selectors?  @gdisf can help! [LINK to meetup]
- Come learn modern CSS3 selectors that will let you write more powerful, specific CSS rules [LINK to meetup]
- Join us to strengthen your CSS rule writing muscles. This week: CSS Selectors [LINK to meetup]
- Not comfortable in the <style> tag? @gdisf to the rescue! [LINK to meetup]

```

**Comment by alfasf** [comment]
got this one!

**Comment by verythorough** [comment]
Closed by #497. Thanks, @alfasf!
